dask/dataframe/io/tests/test_csv.py

```diff
@@ -1529,6 +1529,26 @@ def test_to_csv_simple():
     assert (result.x.values == df0.x.values).all()
 
 
+def test_to_csv_with_single_file_and_append_mode():
+    df0 = pd.DataFrame(
+        {"x": ["a", "b", "c", "d"], "y": [1, 2, 3, 4]},
+    )
+    df = dd.from_pandas(df0, npartitions=2)
+    with tmpdir() as dir:
+        csv_path = os.path.join(dir, "test.csv")
+        with open(csv_path, mode="w") as file:
+            file.write("x, y\n1, 3\n")
+        df.to_csv(
+            str(csv_path),
+            mode="a",
+            header=False,
+            index=False,
+            single_file=True,
+        )
+        result = dd.read_csv(os.path.join(dir, "*"), skiprows=[1]).compute()
+    assert (result.x.values == df0.x.values).all()
```

Comment:
Using `assert_eq` is the recommended way of checking for the correct results.
```suggestion
    assert assert_eq(result, df0)
```

Reply:
(this will fail at the moment because the manually written CSV header includes whitespace " " after the comma)